app/views/forums/_form.html.erb

```diff
@@ -1,27 +1,23 @@
-<%= form_with(model: post) do |form| %>
-  <% if post.errors.any? %>
+<%= form_with(model: forum) do |form| %>
+  <% if forum.errors.any? %>
     <div style="color: red">
-      <h2><%= pluralize(post.errors.count, "error") %> prohibited this post from being saved:</h2>
+      <h2><%= pluralize(forum.errors.count, "error") %> prohibited this forum from being saved:</h2>
 
       <ul>
-        <% post.errors.each do |error| %>
+        <% forum.errors.each do |error| %>
           <li><%= error.full_message %></li>
         <% end %>
       </ul>
     </div>
   <% end %>
 
   <div>
-    <%= form.label :title, style: "display: block" %>
-    <%= form.text_field :title %>
+    <%= form.label :forum_name, style: "display: block" %>
+    <%= form.text_field :forum_name %>
   </div>
 
   <div>
-    <%= form.label :content, style: "display: block" %>
-    <%= form.text_area :content,  size: "50x20" %>
-  </div>
-
-  <div>
-    <%= form.submit %>
```

Comment:
This isn't the main objective of this lesson so I'll let it slide, but without a submit button, users can no longer create or update forums. That would be quite a problem in a real app.